aiohasupervisor/models/backups.py

```diff
@@ -45,6 +45,14 @@ class BackupContent(ResponseData):
     folders: list[Folder]
 
 
+@dataclass(frozen=True)
+class BackupLocationAttributes(ABC):
+    """BackupLocationAttributes model."""
+
+    protected: bool
+    size_bytes: int
+
+
 @dataclass(frozen=True)
 class BackupBaseFields(ABC):
```

Comment:
In a follow-up PR, I think we should remove the deprecated fields from this and other backup related types.
We can't easily change the supervisor HTTP API, but we don't need to include the extra data here.